foundationdb/src/tuple/mod.rs

```diff
@@ -196,4 +238,39 @@ mod tests {
 
         assert_eq!("string".encode_to_vec(), ("string",).encode_to_vec());
     }
+
+    #[test]
+    fn test_recursive_tuple() {
```

Comment:
I think this test shows that #49 is already ok, @yjh0502

Reply:
There's an error on the testcase. Here's an output from python fdb. Please ignore a header value of strings, because python2 string is encoded into binary in fdb bindings.
```
>>> [ord(x) for x in fdb.tuple.pack( ("one", ("two", 42)) )]
[1, 111, 110, 101, 0, 5, 1, 116, 119, 111, 0, 21, 42, 0]
```

There's one more edge case on nested tuple: we should handle nested Empty value differently with top-level empty value.
```
>>> [ord(x) for x in fdb.tuple.pack( (None, (None, None)) )]
[0, 5, 0, 255, 0, 255, 0]
```